index.js

```diff
@@ -651,7 +651,7 @@ class UserAgent {
      * Skeleton method for making a chakram request.
      *
      * @param {function} chakramMethod Chakram request method to call.
-     * @param {array} args Arguments to call the chakram request method with.
+     * @param {Array} args Arguments to call the chakram request method with.
```

Comment:
This is a private method, which is why I didn't have to regenerate the docs for this file.